app/scripts/modules/core/src/deploymentStrategy/strategies/redblack/AdditionalFields.tsx

```diff
@@ -1,95 +1,79 @@
 import React from 'react';
-import { set } from 'lodash';
 
 import { IDeploymentStrategyAdditionalFieldsProps } from '../../deploymentStrategy.registry';
 import { HelpField } from 'core/help/HelpField';
-
 import { IRedBlackCommand } from './redblack.strategy';
 
 export interface IRedBlackStrategyAdditionalFieldsProps extends IDeploymentStrategyAdditionalFieldsProps {
   command: IRedBlackCommand;
 }
 
-export class AdditionalFields extends React.Component<IRedBlackStrategyAdditionalFieldsProps> {
-  private handleChange = (key: string, value: any) => {
-    set(this.props.command, key, value);
-    this.forceUpdate();
-  };
```

Comment:
This is the main thing I wanted to get rid of, but it gets lost in the refactor. Now, the field `onChange` just directly uses the `onChange` prop.